app/services/data_set_documentation.rb

```diff
@@ -8,7 +8,10 @@ def self.for(klass)
       return false
     end
 
-    common_columns = YAML.load_file(Rails.root.join('app/exports/_common_columns.yml'))
+    common_columns = Dir[Rails.root.join('app/exports/common_columns/*')]
+                         .map { |file| YAML.load_file(file) }
+                         .reduce({}, :merge)
```

Comment:
Is it worth pulling this out into a private class method now?

Reply:
Let's think about that when I do the larger refactor of this class once all the export docs are done.